app/src/main/res/layout/fragment_alert_allergens.xml

```diff
@@ -1,57 +1,117 @@
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<?xml version="1.0" encoding="utf-8"?>
+
+<android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:fab="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    xmlns:tools="http://schemas.android.com/tools">
+    android:layout_height="match_parent">
 
-    <LinearLayout
-        android:id="@+id/emptyAllergensView"
+
+    <Button
+        android:id="@+id/btn_add"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_centerVertical="true"
-        android:layout_margin="16dp"
-        android:orientation="vertical"
-        android:padding="16dp">
+        android:layout_gravity="top|center"
+        android:layout_marginHorizontal="10dp"
+        android:clickable="true"
+        android:drawableStart="@drawable/plus_blue"
+        android:paddingStart="20dp"
+        android:paddingEnd="20dp"
+        android:drawablePadding="-20dp"
+        android:focusable="true"
+        android:layout_marginTop="10dp"
+        android:text="Add an allergen"
```

Comment:
Can you extract the string ?

Reply:
To be honest, I'd like to apply this to as many screens as possible.

Reply:
@teolemon I am not sure how to do that , but i can try my best.

Reply:
@deniger @AntonisFaros let's merge as it, then